spec/helpers/application_helper_spec.rb

```diff
@@ -94,4 +94,27 @@
       end
     end
   end
+
+  describe 'page title helper' do
+    subject { helper.page_title }
+
+    context 'when the page title is not set' do
+      it { is_expected.to eq(t('layout.coursemology')) }
+    end
+
+    context 'when the page title is explicitly set' do
+      let(:test_title) { 'this is a test' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.